### PR TITLE
fix(curriculum): Clarified Description

### LINKED ISSUE #47197: Clarify instructions for Cat Photo App Step 1
### Describe the Issue

Been seeing this more lately where people are changing the tag names to `CatPhotoApp` instead of the text. I'm tempted to believe it is because the explanation of where the text for the element is located is at the very bottom of the description, after the actual instructions, and perhaps people just aren't reading that far.

### Affected Page

https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-html-by-building-a-cat-photo-app/step-1

### Your code

Seeing more of this lately in the forums:

```
<html>
<body>
  <catphotoapp>HelloWorld</catphotoapp>
</body>
</html>
```

### Expected behavior

Move explanation about where text is located in element before instructions. My suggestion:

> HTML elements have opening tags like `<h1>` and closing tags like `</h1>`. The text for an element goes between its opening and closing tags.

> Find the `h1` element and change its text to:

> `CatPhotoApp`

Also, I think the intro for this course on the RWD main page should be updated to use the terminology "HTML elements" instead of "HTML tags" so that it is consistent with step 1.

> HTML elements give a webpage its structure. You can use HTML elements to add photos, buttons,  and other elements to your webpage.

> In this course, you'll learn the most common HTML elements by building your own cat photo app.

This makes a much better transition into the first step when we immediately introduce the first HTML element.

### Screenshots

_No response_

### System

N/A

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by hanswang123456**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #47197

By condensing the description, the campers will be explicitly told to put the words "CatPhotoApp" between the closing tags.

### CODE REVIEW

File: curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dc174fcf86c76b9248c6eb2.md [hunk 1]
```diff
@@ -8,12 +8,11 @@ dashedName: step-1
...
 
 HTML elements have opening tags like `<h1>` and closing tags like `</h1>`.
+The text for an element goes between its opening and closing tags.
 
 Find the `h1` element and change its text to:
 
- `CatPhotoApp`
-
-Make sure that the text is between its opening and closing tags.
+CatPhotoApp
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dc174fcf86c76b9248c6eb2.md:15]:
```suggestion
`CatPhotoApp`
```

This does still need to be in back ticks.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

**Review by jeremylt — DISMISSED** [review]

File: curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dc174fcf86c76b9248c6eb2.md [hunk 2]
```diff
@@ -8,12 +8,11 @@ dashedName: step-1
 # --description--
 
 HTML elements have opening tags like `<h1>` and closing tags like `</h1>`.
+The text for an element goes between its opening and closing tags.
```
Comment (ieahleen) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dc174fcf86c76b9248c6eb2.md:11]: If you want to have a new line here, you need to have one empty line between the paragraphs

Comment (hanswang123456) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dc174fcf86c76b9248c6eb2.md:11]: That does look better. Ill commit with changes

**Review by naomi-lgbt — APPROVED** [review]
Okay, I think this is ready!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Comment by hutong9** [comment]
您好：    双换感谢您的来信，收到信后我会第一时间回复您，如需紧急处理请致电15809253357，谢谢！

**Review by jeremylt — APPROVED** [review]

**Pull request merged**